meta/main.yml

```diff
@@ -0,0 +1,16 @@
+---
+galaxy_info:
+  author: "Maor Lipchuk"
+  description: "oVirt Ansible roles"
```

Comment:
description: "Disaster recovery for oVirt engine."
or anything what describes the actual role

Reply:
Done